charts/zookeeper-operator/values.yaml

```diff
@@ -57,6 +57,10 @@ tolerations: []
 annotations: {}
 
 hooks:
+  ## Optionally specify an array of imagePullSecrets. Will override the global parameter if set
```

Comment:
Please update the comment to explain that the array here is expected to contain secret names only, as we seem to have two different schemas already in the code:
- for `pod.imagePullSecrets` we are expecting key-value pair objects
https://github.com/pravega/zookeeper-operator/blob/master/charts/zookeeper/templates/zookeeper.yaml#L110
- for `serviceAccount.imagePullSecrets` we are expecting secret names
https://github.com/pravega/zookeeper-operator/blob/master/charts/zookeeper-operator/templates/service_account.yaml#L7
Otherwise I think it is good to go, unless you would think it necessary to update the schema for `pod.imagePullSecrets` as well (to be able to consume `global.imagePullSecrets` there).
@anishakj would you agree that updating `pod.imagePullSecrets` schema would be appropriate?

Reply:
I tried to be consistent and matched what is already done in the `values.yaml`. See https://github.com/pravega/zookeeper-operator/blob/master/charts/zookeeper-operator/values.yaml#L30. If you like, I could update the whole chart in order to improve this.